packaging/hudi-spark-bundle/pom.xml

```diff
@@ -92,6 +92,7 @@
                   <include>org.apache.httpcomponents:fluent-hc</include>
                   <include>org.antlr:stringtemplate</include>
                   <include>org.apache.parquet:parquet-avro</include>
+                  <include>org.apache.avro:avro</include>
```

Comment:
@xushiyan @alexeykudinkin could you guys check if adding avro dependency is necessary?

Reply:
@Baibaiwuguo can you please explain why do you need an avro bundled into Hudi instead of relying on Spark provided one?